### PR TITLE
OCAPI React App

### PR DESCRIPTION
**Pull request by VelislavaC4Nexus**

### CODE REVIEW

File: my-app/src/cart/Cart.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+const Cart = () => {
+    const { cart } = useCartContext();
+    const navigate = useNavigate();
+    const goToCheckoutHandler = () => {
+        navigate('/checkout');
+    };
+    if (cart && cart.product_items) {
+        return <>
+            <h3 className="fw-bold">Your Cart</h3>
```
Comment (kalipiev) [inline my-app/src/cart/Cart.js]: Try not using hard-coded strings.

File: my-app/src/contexts/CartContext.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { createContext, useState, useContext } from "react";
+
+const CartContext = createContext();
+
+export const useCartContext = () => useContext(CartContext);
+
+export const CartProvider = ({ children }) => {
+    const [cart, setCart] = useState({});//JSON.parse(localStorage.getItem('basket'))
```
Comment (kalipiev) [inline my-app/src/contexts/CartContext.js]: let's remove commented code

File: my-app/src/hooks/useAuth.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import { useState, useEffect } from 'react';
+
+const optionsAuth = {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+    'x-dw-client-id': `aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa`,
```
Comment (kalipiev) [inline my-app/src/hooks/useAuth.js]: client ID should be in a config.

Comment (VelislavaC4Nexus) [reply my-app/src/hooks/useAuth.js]: fixed

File: my-app/src/contexts/CartContext.js (hunk 2 above)
Comment (VelislavaC4Nexus) [reply my-app/src/contexts/CartContext.js]: fixed

File: my-app/src/cart/Cart.js (hunk 1 above)
Comment (VelislavaC4Nexus) [reply my-app/src/cart/Cart.js]: fixed

**Pull request merged**